readme.md

```diff
@@ -1,157 +1,138 @@
-# SQLServer Native Data Connector
-
-## Introduction
-
-This a very work in progress Native Data Connector for SQLServer made in the
-Hasura Hackathon August 2023. It is a fork of
-[postgres-ndc](https://github.com/hasura/postgres-ndc) and aims to follow the
-conventions established there.
-
-Things we have:
-
-- basic queries
-- filtering
-- basic ordering
-- relationships
-- variables
-- sorting
-
-Things we definitely don't have:
-
-- database introspection
-- explain queries
-- reinstate benchmarks
-
-The best view of progress is probably `/crates/ndc-sqlserver/tests/`, and look
-at which tests are still commented out. If you'd to contribute, a very good
-start would be to uncomment one and try to fix any query errors.
-
-## Build
-
-### Prequisites
-
-1. Install [rustup](https://www.rust-lang.org/tools/install).
-2. Install additional tools:
-   - `cargo install cargo-watch cargo-insta`
-   - `rustup component add rust-analyzer`
-   - `rustup component add clippy`
-   - `rustup component add rustfmt`
-3. Install [just](https://github.com/casey/just)
-4. Install [Docker](https://www.docker.com/)
-5. Install protoc. Here are a few options:
-   - `brew install protobuf`
-   - `apt-get install protobuf-compiler`
-   - `dnf install protobuf-compiler`
-6. Clone [v3-engine](https://github.com/hasura/v3-engine) in a directory near this one:
-   ```
-   (cd .. && git clone git@github.com:hasura/v3-engine.git)
-   ```
+# SQL Server Connector
 
-### Compile
+[![Docs](https://img.shields.io/badge/docs-v3.x-brightgreen.svg?style=flat)](https://hasura.io/docs/3.0/getting-started/overview/)
+[![ndc-hub](https://img.shields.io/badge/ndc--hub-sqlserver-blue.svg?style=flat)](https://hasura.io/connectors/sqlserver)
+[![License](https://img.shields.io/badge/license-Apache--2.0-purple.svg?style=flat)](LICENSE.txt)
+[![Status](https://img.shields.io/badge/status-alpha-yellow.svg?style=flat)](./readme.md)
 
-```
-cargo build
-```
+The Hasura SQLServer Connector allows for connecting to a SQL Server database giving you an instant GraphQL API on top
```

Comment:
With this connector, Hasura allows you to instantly create a real-time GraphQL API on top of your data models in Microsoft SQL Server. This connector supports SQL Server's functionalities listed in the table below, allowing for efficient and scalable data operations. Additionally, users benefit from all the powerful features of Hasura’s Data Delivery Network (DDN) platform, including query pushdown capabilities that delegate query operations to the database, thereby enhancing query optimization and performance.

Reply:
Updated